Makefile

```diff
@@ -618,7 +624,7 @@ libcryptopp: $(CRYPTOPP_DIR)/Makefile
 
 .PHONY: libjemalloc
 libjemalloc: $(JEMALLOC_DIR)/Makefile
-	@$(MAKE) --no-print-directory -C $(JEMALLOC_DIR) build_lib_static
+	@$(MAKE) --no-print-directory -C $(JEMALLOC_DIR) build_lib_static | grep -v "Nothing to be done" || true
```

Comment:
Oh, hmm, this looks like it'll suppress real errors from the `make` invocation. Perhaps:

```
@set -o pipefail; $(MAKE) --no-print-directory -C $(JEMALLOC_DIR) build_lib_static | { grep -v "Nothing to be done" || true; }
```